reports/performance/_model.qmd

```diff
@@ -1258,3 +1258,126 @@ model_big_misses_assessment %>%
 ```
 
 :::
+
+## Variance Over Time
+
+These charts look at the variance of the sale price and estimated FMV over time. Ideally, these should remain stable. One should be aware that while ratios may demonstrate consistent trends, because housing values tend to increase over time, the aggregate difference may increase.
+
+::: {.panel-tabset}
+
+
+```{r _organize_variance_data}
```

Comment:
**issue (blocking):** Let's follow the chunk naming convention used throughout the performance report i.e.:

```suggestion
```{r _model_organize_variance_data}
```